api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -303,9 +304,14 @@ public void addLiteralToForm(EditConfigurationVTwo editConfig, FieldVTwo field,
 					String rangeLang = field.getRangeLang();  //UQAM  Default value
 					try {
 						if (_vreq != null ) {
-							rangeLang = _vreq.getLocale().getLanguage();
-							if (!_vreq.getLocale().getCountry().isEmpty()) {
-								rangeLang += "-" + _vreq.getLocale().getCountry();
+							if (editConfig.getLiteralsInScope().get("label").get(0).getLanguage() != "")
```

Comment:
Instead of `!=`, should likely use `!...equals()`.
..or `.isEmpty()`.

Is there a chance that the language could be null? If so, maybe `StringUtils.isBlank()`.

Reply:
good point, i changed it. I don't know if it could be null, so better catch it.